### DISCUSSION TITLE
Setup Issues - A preview release must have at least one semantic commit

### DISCUSSION THREAD

**Discussion by WHenderson** [prisma-labs/dripip#119] [Q&A]
Hi, I believe I've got dripip setup correctly (via this [template](https://github.com/jasonkuhrt/template-typescript-lib)) but I keep getting this error - [A preview release must have at least one semantic commit](https://github.com/WHenderson/immer-store/runs/2574719617?check_suite_focus=true)

What specifically is a semantic commit?

You can take a look at the project [here](https://github.com/WHenderson/immer-store) if it helps.

Thanks in advance,
Will

**Comment by emilyilzrih761-ops** [comment]
Ein „semantic commit“ ist ein Commit, dessen Nachricht einem definierten Format wie Conventional Commits folgt. Zum Beispiel:

feat: add filtering support
fix: resolve validation error
docs: update README

feat, fix, docs usw. geben an, welche Art von Änderung vorgenommen wurde. Für einen Preview-Release benötigt dripip offenbar mindestens einen solchen Commit, um die Versionsänderung https://bizzo-kasino.de/ automatisch bestimmen zu können. Die Seite ist übersichtlich gestaltet und bietet genügend Abwechslung, um auch mal eine entspannte Spielrunde interessant zu gestalten. Werfen Sie vor der Anmeldung einen kurzen Blick auf die Nutzungsbedingungen.
